packages/react-native-client/README.md

```diff
@@ -0,0 +1,402 @@
+# react-native-client
+
+react-native-client is a React Native wrapper for
+[android-client](https://github.com/fishjam-dev/android-client-sdk) and
+[ios-client](https://github.com/fishjam-dev/ios-client-sdk). It allows you to
+quickly and easily create a mobile client app in React Native for a
+[Fishjam](https://github.com/fishjam-dev/fishjam) server.
+
+# Documentation
+
+API documentation is available
+[here](https://fishjam-cloud.github.io/mobile-client-sdk/)
```

Comment:
I think we could link directly to RN documentation:
https://fishjam-cloud.github.io/mobile-client-sdk/modules/_fishjam_cloud_react_native_client.html